### ISSUE TITLE
Could not refresh JMS connection

### ISSUE THREAD

**Issue by jpotts** [Alfresco/alfresco-sdk#616]
Created a new AIO project from the 4.2.0-SNAPSHOT archetype.

Ran ./run.sh build_start which compiled the project and launched Docker containers successfully.

Can log in to Share and browse the Sample Site.

However, preview is not working for DOC, PPT, or XLS files.

In addition, I cannot create a new site. After filling out the site details I get a dialog saying, "03060008 Could not send message".

In the logs, there is a constant stream of messages that look like this:

2021-04-06 19:48:58,124  ERROR [component.jms.DefaultJmsMessageListenerContainer] [Camel (alfrescoCamelContext) thread #2 - JmsConsumer[org.alfresco.transform.781f8366-7cdd.t-reply]] Could not refresh JMS Connection for destination 'org.alfresco.transform.781f8366-7cdd.t-reply' - retrying using FixedBackOff{interval=5000, currentAttempts=93, maxAttempts=unlimited}. Cause: Error while attempting to add new Connection to the pool; nested exception is javax.jms.JMSException: Could not create Transport. Reason: java.io.IOException: Transport scheme NOT recognized: [vm]

I assume that the preview, the inability to create a site, and the log message may all be related to ActiveMQ.

**Comment by jpotts** [comment]
Also, the above was with an out-of-the-box SDK project, so it was running Community Edition.

I get the exact same result if I modify the pom.xml to use:
        
        <alfresco.groupId>org.alfresco</alfresco.groupId>
        <alfresco.bomDependencyArtifactId>acs-packaging</alfresco.bomDependencyArtifactId>
        <alfresco.platform.version>7.0.0</alfresco.platform.version>
        <alfresco.share.version>7.0.0</alfresco.share.version>

        
        <docker.acs.image>quay.io/alfresco/alfresco-content-repository</docker.acs.image>
        <docker.share.image>alfresco/alfresco-share</docker.share.image>

And point my private Alfresco artifacts repo at an ID for which I have credentials.

In other words, it fails on both Community and Enterprise Editions.

**Comment by jpotts** [comment]
The fix is to add:
messaging.subsystem.autoStart=false

To:
platform-docker/src/main/docker/alfresco-global.properties
